content/docs/yaml-reference/assets/index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Assets YAML Runner"
+description: "Asset Actions Used with the Yaml Runner"
+lead: "Asset Actions Used with the Yaml Runner"
+date: 2021-06-09T11:39:03+01:00
+lastmod: 2021-06-09T11:39:03+01:00
+draft: false
+images: []
+menu: 
+  docs:
+    parent: "yaml-reference"
+weight: 202
+toc: true
+---
+
+{{< note >}}
+**Note:** To use the YAML Runner you will need to install the `jitsuin-archivist` python package.
+
+[Click here](https://python.rkvst.com/runner/index.html) for installation instructions.
+{{< /note >}}
+
+## Assets Create
+
+Adding an `asset_label` allows your asset to be referenced in later steps of the story. For example, if you want to add a compliance policy for the asset after it is created.
+
+The `arc_namespace` (for the asset) and the `namespace` (for the location) are used to distinguish between assets and locations created between runs of the story. Usually these field values are derived from an environment variable `ARCHIVIST_NAMESPACE` (default value is namespace).
+
+The optional `confirm: true` entry means that the YAML Runner will wait for the asset to be confirmed before moving on to the next step. This is beneficial if the asset will be referenced in later steps.
+
+For example:
+
+```yaml
+---
+steps:
+  - step:
+      action: ASSETS_CREATE
+      description: Create new EV Pump with id 1.
+      asset_label: ev pump 1
+    behaviours:
+      - Attachments
+      - RecordEvidence
+    attributes:
+      arc_display_name: ev pump 1
+      arc_display_type: pump
+      arc_namespace: wipp
+      ev_pump: "true"
+    confirm: true
+```
+
+The output of this action would be:
```

Comment:
We should explain that this resulting output is visible in the UI

Reply:
added note with hyperlink to the UI